client/src/components/layouts/default.tsx

```diff
@@ -175,7 +175,7 @@ function DefaultLayout({
               name: 'description',
               content: t('metaTags:description')
             },
-            { name: 'keywords', content: t('metaTags:keywords') }
+            { name: 'keywords', content: t('metaTags:keywords').join() }
```

Comment:
This is false positive type, the value should be correct, but the type is the wrong one. it will work with array without issue

You will need to import `DefaultTFuncReturn` tho

```suggestion
            { name: 'keywords', content: t<string, DefaultTFuncReturn & string[]>('metaTags:keywords') }
```